[31mERRO[0m[2025-03-27 15:53:15]main.go:147 main.main() Database ping error                           [31mdatabase[0m="postgres://postgres:haron@localhost:5432/Cdrrmo?sslmode=disable" [31merror[0m="failed to ping DB (database: postgres://postgres:haron@localhost:5432/Cdrrmo?sslmode=disable): pq: database \"Cdrrmo\" does not exist"
[36mINFO[0m[2025-03-27 15:55:57]main.go:150 main.main() Successfully connected to database            [36mdatabase[0m="postgres://postgres:haron@localhost:5432/Cdrrmo?sslmode=disable"
[36mINFO[0m[2025-03-27 15:55:57]main.go:167 main.main() Migrations applied successfully (or no changes needed)  [36mmigrationsPath[0m="file://../../internal/migrations"
[36mINFO[0m[2025-03-27 15:55:57]main.go:271 main.main() Starting server                               [36mport[0m=8080
